### ISSUE TITLE
How to get the FLOPs of a model

### ISSUE THREAD

**Issue by zheng-ningxin** [google-research/nasbench#20]
Hi~ 
   Is there any way to get the FLOPs of an existing model in the search space?
Thanks~

**Comment by AlbertiPot** [comment]
Yes, the FLOPs are vital properties for DNN. Is there any function to help to calculate the subnets` FLOPs

**Comment by ngctnnnn** [comment]
In NasBench201, the authors have proposed this tool to measure the FLOPs from a constructed model that is found in the Bench.
https://github.com/D-X-Y/AutoDL-Projects/blob/d04edcd211604e3bb3e682ba5606c0c0c5e991ce/xautodl/utils/flop_benchmark.py
